### PR TITLE
corectl 0.7.9

### PR DESCRIPTION
**Pull request by AntonioMeireles**
please bump. 

thanks!

### CODE REVIEW

**Comment by AntonioMeireles** [comment]
@ilovezfs if awke ;-) && thanks in advance!

**Comment by ilovezfs** [comment]
>&& thanks in advance!

Does that mean that I *get no thanks* if I'm not awake?

File: Formula/corectl.rb [hunk 1]
```diff
@@ -42,6 +42,7 @@ def install
 
       bin.install Dir["bin/*"]
 
+      doc.install "README.md", "LICENSE"
```
Comment (ilovezfs) [inline Formula/corectl.rb]: @AntonioMeireles This should not be here. Those go in the prefix and are handled automatically: https://github.com/Homebrew/brew/blob/master/Library/Homebrew/metafiles.rb

**Comment by AntonioMeireles** [comment]
Of course not, You all deserve many thanks for keeping this alive... It's
just that you seem to be in a timezone similar to mine so being you to push
would reduce latency 😀

File: Formula/corectl.rb (hunk 1 above)
Comment (ilovezfs) [inline Formula/corectl.rb]: @AntonioMeireles ping

Comment (AntonioMeireles) [inline Formula/corectl.rb]: missed this earlier. done. && THANKS!

**Comment by AntonioMeireles** [comment]
@ilovezfs (pong). build passed ok, anything i've still missed or you just busy. 

thanks in advance

**Comment by AntonioMeireles** [comment]
@ilovezfs  you 🚀 !

**Comment by ilovezfs** [comment]
@AntonioMeireles Keep up the great work! By the way have you seen our new formula? https://github.com/Homebrew/homebrew-core/blob/master/Formula/govendor.rb

**Pull request closed without merging**